app/services/rates_importer_service.rb

```diff
@@ -25,14 +25,16 @@ def import(row, rating_year, council)
         suburb: suburb,
         town_city: town_city,
         rating_year: rating_year,
+        include_in_address_lookups: true,
         meta: row.to_s
       )
+    else property.update(include_in_address_lookups: true)
     end
     rates_bill = RatesBill.find_by(property: property, rating_year: rating_year)
 
     if rates_bill.present?
-      current_rates = rates_bill.total_rates.to_f.round(2)
-      new_rates = (total_rates.to_f + total_water_rates.to_f).round(2)
+      # current_rates = rates_bill.total_rates.to_f.round(2)
+      # new_rates = (total_rates.to_f + total_water_rates.to_f).round(2)
```

Comment:
Why are these lines commented out? Should they be deleted entirely if they aren't needed?